Eshop.WebApi/Controllers/CategoryController.cs

```diff
@@ -16,38 +24,46 @@ public class CategoryController : Controller
         [HttpGet]
         public List<Category> GetCategories()
         {
-            return Categories;
+            dbContext.SaveChanges();
+
+            return dbContext.Categories.ToList(); ;
         }
 
         [HttpGet("{id}")]
         public Category GetCategory(int id) 
-        { 
-            return Categories.First(x => x.Id == id);
+        {
+            dbContext.SaveChanges();
```

Comment:
remove this - I said only when you do mutations not queries - GET is query, POST, PUT, PATCH, DELETE are mutations

Reply:
removed